src/modules/pf3dTracker/include/iCub/pf3dTracker.hpp

```diff
@@ -59,16 +64,10 @@
 using namespace std;
```

Comment:
@gsaponaro Please remove all `using namespace ..` from the header files. Use them only in cpp.